docs/maintainers/style-guide.md

```diff
@@ -110,22 +110,26 @@ All the above will greatly help reduce churn in reviews, whch is frustrating for
 
 There are a couple of ways in which you can format Markdown documents using prettier:
 
-#### 1. Using editor integration
+#### 1. Using a GitHub comment
 
-Prettier can be integrated with many [editors][prettier-editor]. This works especially well if the editor supports automatic formatting on save.
+If you add a comment to a GitHub PR that contains the text `/format`, a GitHub workflow will format all Markdown and JSON documents using prettier. Any formatting changes made by prettier will automatically be committed to the PR's branch. This also works for forks that have [enabled maintainers to edit the fork's PR][allowing-fork-pr-changes].
```

Comment:
Some tracks like JS/TS also use prettier to format their docs but mentioning that here seems like it'll only cause confusion, so I left it out.